ironfish-cli/src/commands/start.ts

```diff
@@ -100,6 +100,18 @@ export default class Start extends IronfishCommand {
       allowNo: true,
       description: 'Run migrations when an upgrade is required',
     }),
+    // TODO: Update description with a list of network IDs for mainet and official testnets.
```

Comment:
```suggestion
    // TODO: Update description with a list of network IDs for mainnet and official testnets.
```

I'm actually okay just dropping this TODO as well

Reply:
Removed